include/autocheck/reporter.hpp

```diff
@@ -92,8 +92,76 @@ namespace autocheck {
       }
   };
 
+  class msg_reporter : public reporter {
```

Comment:
What's going on here?

I think msg_reporter can be removed and gtest_checker can use ostream_reporter with ostringstream.